src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx

```diff
@@ -44,6 +86,45 @@ const MethodologySummaryPage = ({
                   'Not yet published'
                 )}
               </SummaryListItem>
+              {currentMethodology.publication && (
+                <SummaryListItem term="Publication">
+                  <Link
+                    to={generatePath<ReleaseRouteParams>(
+                      releaseSummaryRoute.path,
+                      {
+                        publicationId: currentMethodology.publication.id,
+                        releaseId: currentMethodology.publication.releaseId,
+                      },
+                    )}
+                  >
+                    {currentMethodology.publication.title}
+                  </Link>
+                </SummaryListItem>
+              )}
+              {currentMethodology.otherPublications &&
+                currentMethodology.otherPublications.length > 0 && (
+                  <SummaryListItem term="Other publications using this methodology">
```

Comment:
This seems like a bit of a mouthful as we're already in the context of a methodology. Would suggest calling this 'Other publications' (combined with the other suggestion for 'Owning publication') instead?